triton/apps/vscode.rst

```diff
@@ -1,13 +1,43 @@
 VSCode on Triton
 ================
 
-VSCode is available through :doc:`../usage/ood`.  If you want to
-connect through your own computer, read below.
+VSCode is available through :doc:`../usage/ood`, and with this you can
+select whatever resources you want and run directly in the Slurm queue
+(and perform your actual calculations).  If you want to
+connect through your own computer, it can provide a nice interface but
+read below.
+
+As always when using user-contributed extensions, be cautious of what
+extensions you install.  A malicious extension can access and/or
+delete all of the data available via your account.
+
+
+
+VSCode through OOD
+------------------
+
+.. seealso:: :doc:`../usage/ood`
+
+This is useful for getting things done quickly, but running in a web
+browser can be limited in some cases.
+
+
+
+VSCode remote SSH
+-----------------
+
+**"Remote SSH"** is a nice way to work on a remote computer and
+provides both editing and shell access, but everything will run
+directly on the login node on Triton.  This is OK for editing, but not
+for main computations (see the section above or below).  **To repeat:
+don't use this for running big computations.**
```

Comment:
Sometimes I get the very good question: How can I find out that I am running a computation on login node?

Reply:
Added a screenshot mentioning this (but I'm not entirely sure it will show the same for everyone!)